docs/contributing/preview-features.md

```diff
@@ -1,14 +1,3 @@
 # Preview Features
 
-If you're running GitHub Desktop you can enable **"preview features"**, which
-we consider something we want to eventually ship to everyone, but needs some
-sort of refinement before we can do that, and in the meantime we would like
-feedback from users on these features.
-
-To opt-in for testing preview features, set the
-`GITHUB_DESKTOP_PREVIEW_FEATURES` environment variable to any value and launch
-the Desktop app.
-
-## Preview Features
-
-Coming Soon™
+We are hard at work building the latest features, we will have something to show soon...
```

Comment:
Let's just remove this whole file (and open an issue to track this idea properly) :fire::fire::fire: